src/character.h

```diff
@@ -241,8 +243,8 @@ class Character : public Creature, public visitable<Character>
         /** Toggles a trait on the player and in their mutation list */
         void toggle_trait(const std::string &flag);
         /** Add or removes a mutation on the player, but does not trigger mutation loss/gain effects. */
-        void set_mutation(const std::string &flag);
```

Comment:
It does, and most of our code would be a lot nicer if it had more dedicated functions and fewer switches.